apps/condo/domains/meter/components/AddressAndUnitInfo.tsx

```diff
@@ -0,0 +1,161 @@
+import { UnitInfo } from '@condo/domains/property/components/UnitInfo'
+import { PropertyFormItemTooltip } from '@condo/domains/property/PropertyFormItemTooltip'
+
+
+const FORM_ROW_MEDIUM_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 20]
+const FORM_ITEM_WRAPPER_COLUMN_STYLE: ColProps = { style: { width: '100%', padding: 0 } }
+
+type PROPERTY_UNIT_INITIAL_VALUES = {
+    propertyId?: string
+    unitName?: string
+    unitType?: string
+}
+
+type ADDRESS_AND_UNIT_INFO_PROPS = {
+    form: FormInstance,
+    organizationId: string,
+    meterType: MeterPageTypes,
+    getHandleSelectPropertyAddress: (form: FormInstance) => (_: any, option: DefaultOptionType) => void,
+    handleDeselectPropertyAddress: () => void
+    selectedPropertyId: string,
+    isMatchSelectedProperty: boolean
+    isNoMeterForAddress?: boolean,
+    notFoundMetersForAddressTooltip?: JSX.Element,
+    isNoMeterForUnitName?: boolean
+    property?: PropertyType,
+    propertyLoading?: boolean
+    setSelectedUnitName?: Dispatch<SetStateAction<string>>,
+    setSelectedUnitType?: Dispatch<SetStateAction<string>>,
+    initialValues?: PROPERTY_UNIT_INITIAL_VALUES
+}
+
+
+export const AddressAndUnitInfo = (props: ADDRESS_AND_UNIT_INFO_PROPS): JSX.Element => {
+    const intl = useIntl()
+    const AddressLabel = intl.formatMessage({ id: 'field.Address' })
+    const AddressPlaceholder = intl.formatMessage({ id: 'placeholder.Address' })
+    const ClientInfoMessage = intl.formatMessage({ id: 'ClientInfo' })
+
+    const {
+        getHandleSelectPropertyAddress,
+        handleDeselectPropertyAddress,
+        selectedPropertyId, isMatchSelectedProperty,
+        organizationId,
+        isNoMeterForAddress,
+        notFoundMetersForAddressTooltip,
+        isNoMeterForUnitName,
+        meterType,
+        property,
+        propertyLoading,
+        setSelectedUnitName,
+        setSelectedUnitType,
+        form,
+        initialValues,
+    } = props
+
+    const { requiredValidator } = useValidations()
+    const { addressValidator } = usePropertyValidations()
+    const validations = {
+        property: [requiredValidator, addressValidator(selectedPropertyId, isMatchSelectedProperty)],
+    }
+
+    return (
+       
+        <Row justify='space-between' gutter={FORM_ROW_MEDIUM_VERTICAL_GUTTER}>
+            <Col span={24}>
+                {meterType !== METER_TAB_TYPES.propertyMeter && (
+                    <Typography.Title level={3}>
+                        {ClientInfoMessage}
+                    </Typography.Title>
+                )}
+            </Col>
+            <Col span={24}>
+                {notFoundMetersForAddressTooltip ? <Tooltip
+                    className='no-content-found-tooltip'
+                    open={isNoMeterForAddress}
+                    title={notFoundMetersForAddressTooltip}
+                    placement='bottomLeft'
+                >
```

Comment:
Related to figma, you need to use dropdown of select here, not `Tooltip`

<img width="848" alt="image" src="https://github.com/open-condo-software/condo/assets/52532264/1f4dbbd8-0c3a-420c-a43f-bc5460cb4eb3">

Reply:
Mb somehow reuse `notFoundContent` prop of `PropertyAddressSearchInput` or move this message out of property and unit fields (for example, in alert below)

Reply:
agreed with design team to use just plain text under the component